Gems/EMotionFX/Code/EMotionFX/CommandSystem/Source/AnimGraphNodeGroupCommands.cpp

```diff
@@ -22,20 +23,34 @@
 
 namespace CommandSystem
 {
+    AZ_CLASS_ALLOCATOR_IMPL(CommandAnimGraphAdjustNodeGroup, EMotionFX::CommandAllocator, 0)
+
     //--------------------------------------------------------------------------------
     // CommandAnimGraphAdjustNodeGroup
     //--------------------------------------------------------------------------------
-    CommandAnimGraphAdjustNodeGroup::CommandAnimGraphAdjustNodeGroup(MCore::Command* orgCommand)
+    CommandAnimGraphAdjustNodeGroup::CommandAnimGraphAdjustNodeGroup(
+        MCore::Command* orgCommand,
+        AZ::u32 animGraphId,
+        AZStd::string name,
+        AZStd::optional<bool> visible,
+        AZStd::optional<AZStd::string> newName,
+        AZStd::optional<AZStd::vector<AZStd::string>> nodeNames,
+        AZStd::optional<NodeAction> nodeAction,
+        AZStd::optional<AZ::u32> color,
+        AZStd::optional<bool> updateUI
```

Comment:
Nit: updateUI -> updateUi

Reply:
hmm, I'd rather keep it updateUI.

Reply:
As far as I am aware this is recommended by our coding standard to use updateUi and not updateUI, even though I am sure you will find places in our code that don't follow that.

Reply:
This is actually more than a recommendation. It is a requirement by the [Coding Standard](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#abbreviations)

Reply:
Well, that's a mess. There's approximately 500 uses of "UI" in the EMotionFX code. Not to mention we have "Scene**API**", "Scene**UI**", "AzToolsFramework/**UI**", and even "**AZStd**". For now, the command's parameter is "updateUI". I think keeping the casing of the command's parameter and the member variable it represents is best. We can do a separate pass to unify that within EMotionFX.

Separately, we can't just change the method names. For instance, `AzToolsFramework::PropertyHandler` has methods called `WriteGUIValuesIntoProperty`, `ReadValuesIntoGUI`, including a parameter name with `GUI`. So getting conformance to that standard will take a while.